src/components/Dialogs/Multistepper/Multistepper.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react'
+import Dialog from '../GeneralDialog/GeneralDialog'
+import { DialogProps } from '@/components/Dialogs/GeneralDialog/GeneralDialog'
+import {
+  StyledHeader,
+  StyledStepsInfoContainer,
+  StyledStepInfo,
+  StyledCircle,
+  StyledStepInfoText,
+  StyledExitButton,
+} from './Multistepper.style'
+import { Icon } from '@/shared/components'
+
+type Step = {
+  title: string
+  element: JSX.Element
+}
+
+type MultistepperProps = {
+  steps: Step[]
+  currentStep?: number
+} & DialogProps
+
+const Multistepper: React.FC<MultistepperProps> = ({
+  steps,
+  currentStep = 0,
+  exitButton,
+  handleExit,
+  ...dialogProps
+}) => {
+  return (
+    <Dialog wide {...dialogProps}>
+      <StyledHeader>
+        <StyledStepsInfoContainer>
+          {steps.map((step, idx) => {
+            const isActive = idx === currentStep
+            const isCompleted = currentStep > idx
+            const isLast = idx === steps.length - 1
+
+            return (
+              <StyledStepInfo key={step.title} isActive={isActive}>
+                <StyledCircle isFilled={isActive || isCompleted}>
+                  {isCompleted ? <Icon name="check" /> : idx + 1}
+                </StyledCircle>
+                <StyledStepInfoText isActive={isActive}>{step.title}</StyledStepInfoText>
+                {isLast ? null : <hr />}
+              </StyledStepInfo>
+            )
+          })}
+        </StyledStepsInfoContainer>
+        {exitButton && (
```

Comment:
Doesn't the base dialog already have this?

Reply:
Positioning differs. `exitButton` in `Multistepper` is aligned with steps' indicators while in base dialog it's positioned with top/right. That's why I decided to overwrite that. Tried using base dialog button at first but didn't come up with something that looks good

Reply:
Hmm yeah I see. That seems like an inconsistency on designers part. I also had to make some tweaks to the exit button for my image crop. I suggest we use exit button from the base dialog for now even if it doesn't look ideal. Hopefully my changes help, if they don't, we will see something's off during integration and can tweak that then.